<details><summary>Linter check found the following problems:</summary>
Traceback (most recent call last):
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/percy/render/_renderer.py", line 232, in render
    return ruamel.load(yaml_text.replace("\t", " ").replace("%", " "))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/main.py", line 434, in load
    return constructor.get_single_data()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/constructor.py", line 119, in get_single_data
    node = self.composer.get_single_node()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/composer.py", line 76, in get_single_node
    document = self.compose_document()
               ^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/composer.py", line 99, in compose_document
    node = self.compose_node(None, None)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/composer.py", line 143, in compose_node
    node = self.compose_mapping_node(anchor)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/composer.py", line 216, in compose_mapping_node
    while not self.parser.check_event(MappingEndEvent):
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/parser.py", line 146, in check_event
    self.current_event = self.state()
                         ^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/parser.py", line 612, in parse_block_mapping_key
    raise ParserError(
ruamel.yaml.parser.ParserError: while parsing a block mapping
  in "<unicode string>", line 4, column 1:
    package:
    ^ (line: 4)
expected <block end>, but found '<block mapping start>'
  in "<unicode string>", line 45, column 2:
     license_family: Other
     ^ (line: 45)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/lint/__init__.py", line 722, in lint_file
    recipe = _recipe.Recipe.from_file(
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/percy/render/recipe.py", line 231, in from_file
    raise exc
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/percy/render/recipe.py", line 222, in from_file
    recipe._load_from_string(text.read())
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/percy/render/recipe.py", line 154, in _load_from_string
    self.render()
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/percy/render/recipe.py", line 271, in render
    self.meta = renderer.render(
                ^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/percy/render/_renderer.py", line 255, in render
    raise YAMLRenderFailure(recipe_dir, line=exc.problem_mark.line)
percy.render.exceptions.YAMLRenderFailure: /tmp/abs_64cg6a74uz/clone/recipe

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/conda/envs/linter/bin/conda-lint", line 11, in <module>
    sys.exit(main())
             ^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/run.py", line 114, in main
    result = linter.lint(
             ^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/lint/__init__.py", line 626, in lint
    msgs = self.lint_file(
           ^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/lint/__init__.py", line 739, in lint_file
    recipe = _recipe.Recipe(recipe_name)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/percy/render/recipe.py", line 93, in __init__
    self.recipe_dir = recipe_file.parent
                      ^^^^^^^^^^^^^^^^^^
AttributeError: 'str' object has no attribute 'parent'

ERROR conda.cli.main_run:execute(49): `conda run conda-lint /tmp/abs_64cg6a74uz/clone` failed. (See above for error)
</details>

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -42,7 +42,7 @@ test:
 about:
   home: https://github.com/zopefoundation/persistent/
   license: BSD-3-Clause AND ZPL-2.1
- license_family: Other
+  license_family: Other
   license_file:
     - COPYRIGHT.txt
     - LICENSE.txt

